demos/launch/simulation.launch.xml

```diff
@@ -42,10 +45,14 @@
         name="IGN_GAZEBO_SYSTEM_PLUGIN_PATH"
         value="$(var plugin_path)"/>
 
-     <env
+      <env
```

Comment:
Nit, remove this change?

Reply:
pls go ahead to remove it. Apparently, the automatic style formatting does not apply to xml.